README.md

```diff
@@ -54,3 +54,190 @@ For a simple, gentler, introduction to Lagom, have a look at the [Hello World gi
 - Getting help
 
 If you have any troubles and need help, feel free to ask in the [Gitter channel](https://gitter.im/lagom/lagom)
+
+## Running: Kubernetes
+
+This project uses [Lightbend's Platform Tooling](https://s3-us-west-2.amazonaws.com/rp-tooling-temp-docs/home.html) to
+simplify deployment to [Kubernetes](https://kubernetes.io/).
+
+### Setup
+
+* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl)
+* [Minikube](https://github.com/kubernetes/minikube) v0.25.0 or later (Verify with `minikube version`)
+* [Helm](https://github.com/kubernetes/helm)
+* [reactive-cli](https://github.com/lightbend/reactive-cli) 0.9.0 or later (Verify with `rp version`)
+
+
+### Deployment Workflow
+
+> Note that this is an alternative to the Operations workflow documented below.
+
+The following workflow is well suited for development purposes. It will build Docker images for each subproject, generate Kubernetes
+YAML, and deploy them into your Minikube using `kubectl`. It also installs the [Reactive Sandbox](https://github.com/lightbend/reactive-sandbox)
+in your Minikube, providing the required dependencies for Kafka, Cassandra, and Elasticsearch.
+
+> Want a fresh Minikube? Run `minikube delete` before the steps below.
+
+`minikube start --memory 6000`
+
+`sbt "deploy minikube"`
+
+> Open the URL this command prints in the browser
+
+```bash
+echo "http://$(minikube ip)"
+```
+
+### Operations Workflow
+
+> Note that this is an alternative to the Development workflow documented above.
+
+This workflow matches more closely how an application would be deployed in a production environment. In this method,
+the images are built by SBT. The [reactive-cli](https://github.com/lightbend/reactive-cli) is then
+used to generate YAML for deployment to the Kubernetes cluster. The steps below focus on Minikube, but could be used on
+a real Kubernetes cluster given access to a Docker registry and configured `kubectl`.
+
+> Want a fresh Minikube? Run `minikube delete` before the steps below.
+
+##### Setup Docker engine context to point to Minikube
+
+```bash
+eval $(minikube docker-env)
+```
```

Comment:
Could add a link "For more information..." that points to https://kubernetes.io/docs/getting-started-guides/minikube/#reusing-the-docker-daemon.